### ISSUE TITLE
"nikola new_post" crashes with Nikola 7.8.15, Python 3.5, logbook 1.1.0, Linux

### ISSUE THREAD

**Issue by hartwork** [getnikola/nikola#3138]
Hi!

Am I doing anything wrong here?
```console
# nikola new_post
Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.5/nikola", line 11, in <module>
    load_entry_point('Nikola==7.8.15', 'console_scripts', 'nikola')()
  File "/usr/lib64/python3.5/site-packages/pkg_resources/__init__.py", line 479, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib64/python3.5/site-packages/pkg_resources/__init__.py", line 2703, in load_entry_point
    return ep.load()
  File "/usr/lib64/python3.5/site-packages/pkg_resources/__init__.py", line 2321, in load
    return self.resolve()
  File "/usr/lib64/python3.5/site-packages/pkg_resources/__init__.py", line 2327, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib64/python3.5/site-packages/nikola/__init__.py", line 35, in <module>
    from .nikola import Nikola  # NOQA
  File "/usr/lib64/python3.5/site-packages/nikola/nikola.py", line 62, in <module>
    from .post import Post  # NOQA
  File "/usr/lib64/python3.5/site-packages/nikola/post.py", line 44, in <module>
    from . import utils
  File "/usr/lib64/python3.5/site-packages/nikola/utils.py", line 49, in <module>
    import logbook
  File "/usr/lib64/python3.5/site-packages/logbook/__init__.py", line 14, in <module>
    from .base import (
  File "/usr/lib64/python3.5/site-packages/logbook/base.py", line 19, in <module>
    from logbook.concurrency import (greenlet_get_ident, thread_get_ident,
  File "/usr/lib64/python3.5/site-packages/logbook/concurrency.py", line 31, in <module>
    from gevent._threading import (Lock as ThreadLock,
ImportError: cannot import name 'RLock'
```

Best, Sebastian

**Comment by hartwork** [comment]
Turned out to be fixed by upgrading to logbook 1.4.0. I'm unsure if `requirements.txt` should be touched. What do you think?

**Comment by ralsina** [comment]
Sounds like it. I'll just move it to 1.4.0
